### DISCUSSION TITLE
Relaxed and hlky?

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#1087] [General]
I was wondering why the alternate ways to test checkpoints are gone and only Automatic1111 remains. I know that is probably because merged checkpoints don't work since the safetensors update but it is also a shame to not have different options more simpler and faster than Automatic1111. Can you bring back relaxed Ben? Thanks.

**Comment by TheLastBen** [comment]
I will make a relaxed version of A1111 later, the old colabs needed a lot of work and I don't have time for them
